baybe/constraints/discrete.py

```diff
@@ -44,18 +49,16 @@ def get_invalid(self, data: pd.DataFrame) -> pd.Index:  # noqa: D102
         res = reduce(_valid_logic_combiners[self.combiner], satisfied)
         return data.index[res]
 
-    def to_polars(self) -> pl.Expr:  # noqa: D102
+    def get_invalid_polars(self) -> pl.Expr:  # noqa: D102
```

Comment:
I'm a bit surprised, tbh. Somehow I have expected that we harmonize interfaced in way that we have both pandas an polars simply indicate the parts that actually *pass* the constraint. Would feel much more natural to me, instead of somehow coding conditions that are then combined to an expression indication what *not* to keep, only then to invert the filter arguments once again 🤔 

So basically `get_valid_pandas` and `get_valid_polars` or something like `is_fulfilled_pandas` ... Or what are your thoughts here?

Reply:
why does that matter its an arbitrary binary choice whether to indicate valid or invalid rows? Since we use `.drop` the invalid ones even make more sense

Reply:
Not saying that it must be changed, just considered the current version less intuitive. Let's close and proceed